roles/fio-bench/tasks/main.yml

```diff
@@ -17,20 +17,20 @@
       kind: Job
       apiVersion: batch/v1
       metadata:
-        name: '{{ meta.name }}-fio-client-{{item}}-bench'
+        name: '{{ meta.name }}-fio-client-{{item}}-benchmark'
         namespace: '{{ meta.namespace }}'
       spec:
         ttlSecondsAfterFinished: 600
         template:
           metadata:
             labels:
-              app: fio-bench
+              app: fio-benchmark
```

Comment:
since this was changed, there's a change required for tests/test_uperf.sh to reflect this

Reply:
No, it doesn't.

Reply:
Since the label was updated we'll still need to update tests/test_uperf.sh to look at app=fio-benchmark instead of fio-bench